target/appengine-staging/WEB-INF/appengine-generated/app.yaml

```diff
@@ -0,0 +1,65 @@
+runtime: java8
```

Comment:
target/ files should not be checked in

Reply:
these were deleted in the PR #33 - similarly to the deleted feedback.java file, I can't pull these into my local editor and they aren't there.